### DISCUSSION TITLE
How to recover after reinstallation ?

### DISCUSSION THREAD

**Discussion by jimbo72** [immich-app/immich#5606] [Q&A] [answered]
My library is on an external disk (on a mount). For some reasons I did a complete reinstallation.
How to recover my files in Immich ? My photos/videos are still on the disk, but immich does not show them.
They are in "untracks files", how to repair that ?

**Comment by bo0tzz** [comment, marked as answer]
Did you keep a copy/backup of the postgres database? You'll need that to recover your installation, otherwise you'll have to start from scratch and upload the files again.

**Comment by jimbo72** [comment]
No, I have no backup of the database, ok I have to start from scratch.
Thanks for the reply
